Makefile

```diff
@@ -58,6 +59,24 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+ifeq ($(ARCH),s390x)
+K8S_VERSION=v1.25.5
```

Comment:
Hi @BbolroC ,

is there a reason to *not* use "v1.24.2" which is the current version used in x86_64 tests?

I checked that kube-apiserver and kubectl 1.42.2 for s390x are available. Not sure about etcd.

Reply:
Ah, I forgot updating them and keeping the versions to those specified in Dockerfile for kubebuilder-tools. I will update them after checking the version of `etcd`. Thanks for pointing it out. :wink:

Reply:
It is `3.5.4`:

```
[~/.local/share/kubebuilder-envtest/k8s/1.24.2-linux-amd64]$ ./etcd --version
etcd Version: 3.5.4
Git SHA: 08407ff76
Go Version: go1.16.15
Go OS/Arch: linux/amd64
```